packages/browser-wallet/src/popup/shared/utils/baking-helpers.ts

```diff
@@ -13,10 +13,13 @@ const fractionResolution = 100000;
 const percentageModifier = fractionResolution / 100;
 
 export const decimalToRewardFraction = (d: number | undefined) =>
-    d === undefined ? undefined : d * percentageModifier;
+    d === undefined ? undefined : Math.floor(d * percentageModifier);
```

Comment:
I'll revert this. I added it because before I allowed too many decimals, which would cause the wallet to crash. But now I actually ensure that we don't set rates with too many decimals, so this is not necessary.